### ISSUE TITLE
SIP packet inspection for informational purposes

### ISSUE THREAD

**Issue by Commifreak** [opnsense/plugins#3918]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [x] When the request is meant for an existing plugin, I've added its name to the title.

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
A plugin for SIP inspection/logging for informational purpose would be nice.

I have a Gigaset Go Box 100 which connects to tel.t-online.de and sends SIP/RTP packets. A plugin which sees those requests and builds some session info history would be awesome.

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by AdSchellevis** [comment]
have you considered using the packet capture (https://docs.opnsense.org/manual/diagnostics_interfaces.html#packet-capture)?

**Comment by Commifreak** [comment]
Yes. But thats for one time usage. I thought of something that do it in nackground amd creates a neat info table (start, end, caller, callee, length, used codec)

**Comment by AdSchellevis** [comment]
call monitoring is more a job for your pbx if you ask me...

**Comment by Commifreak** [comment]
True. Its not capable of this at all, but you are right.

well, that was just an idea 😀
